src/client/Microsoft.Identity.Client/Internal/Requests/Interactive/AuthCodeExchangeComponent.cs

```diff
@@ -59,5 +60,27 @@ public Task<MsalTokenResponse> FetchTokensAsync(CancellationToken cancellationTo
 
             return dict;
         }
+
+        private void AddCCSHeadersToTokenClient()
```

Comment:
nit: use the ClientInfo class, so you don't have to worry about error handling w/the json parsing. 
`ClientInfo.CreateFromJson(authResult.ClientInfo);`